.github/workflows/guideline-themes-continuous-integration.yml

```
@@ -0,0 +1,73 @@
+name: Guideline Themes CI
+
+on:
+    pull_request:
+        paths:
+            - packages/guideline-themes-settings/**
+
+# Ensures that only one workflow per branch will run at a time.
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+jobs:
+    lint-typecheck:
+        runs-on: ubuntu-latest
+        timeout-minutes: 10
+
+        strategy:
+            fail-fast: false
+
+        steps:
+            - name: Checkout default branch
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 0
+
+            - name: Use pnpm
+              uses: pnpm/action-setup@v3
+              with:
+                  run_install: false
+
+            - name: Use Node.js
+              uses: actions/setup-node@v4
+              with:
+                  node-version-file: ".nvmrc"
+                  cache: "pnpm"
+
+            - name: Install npm dependencies
+              run: pnpm i --frozen-lockfile
+
+            - name: Typecheck code
+              run: pnpm --stream --filter {packages/guideline-themes-settings} typecheck
+
+            - name: Lint code
+              run: pnpm --stream --filter {packages/guideline-themes-settings} lint
+
+    unit-tests:
+        runs-on: ubuntu-latest
+        timeout-minutes: 10
+
+        strategy:
+            fail-fast: false
+
+        steps:
+            - name: Checkout current commit
+              uses: actions/checkout@v4
+
+            - name: Use Node.js
+              uses: actions/setup-node@v4
+              with:
+                  node-version-file: ".nvmrc"
+                  cache: "pnpm"
+
+            - name: Use pnpm
+              uses: pnpm/action-setup@v3
+              with:
+                  run_install: false
```

They need to be swapped
```suggestion
            - name: Use pnpm
              uses: pnpm/action-setup@v3
              with:
                  run_install: false
                  
            - name: Use Node.js
              uses: actions/setup-node@v4
              with:
                  node-version-file: ".nvmrc"
                  cache: "pnpm"
```